### PR TITLE
Style the grid container

### PR DESCRIPTION
**Pull request by andreeanica16**

### CODE REVIEW

**Comment by andreeanica16** [comment]
@tblanshard @iht @juliocc

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -203,11 +190,16 @@ tr:nth-child(even) {
 .gridElement {
   display: grid;
   grid-template-columns: repeat(2, 50%);
-  grid-template-rows: auto auto auto auto auto;
+  grid-template-rows: auto auto 250px auto auto;
```
Comment (tblanshard) [inline src/main/webapp/style.css:193]: As discussed, I think the fourth column should be fixed due to the addition of subtitles.
```suggestion
  grid-template-rows: auto auto auto 250px auto;
```

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -273,7 +282,6 @@ tr:nth-child(even) {
 .sdkAnalysis {
   grid-column: 1;
   grid-row: row 3;
-  background-color: papayawhip;
```
Comment (tblanshard) [inline src/main/webapp/style.css:276]: Would it be possible not to delete these just yet - at the moment I'm using them for testing as I add graphs (just so I can see what is working and what isn't as the graphs show up clearer against a coloured background) - thanks!

File: pom.xml [hunk 3]
```diff
@@ -110,14 +110,6 @@
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
-      <plugin>
```
Comment (tblanshard) [inline pom.xml:113]: Not sure if this code affects your tests, but my test for checking the permissions are set up correctly needs this to find the JSON API key file?

File: src/main/webapp/style.css (hunk 2 above)
Comment (andreeanica16) [reply src/main/webapp/style.css:276]: Sure! No problem :D

Comment (andreeanica16) [reply src/main/webapp/style.css]: solved here https://github.com/googleinterns/step229-2020/pull/47/commits/794ab204e3639442e7688891bc2f6fd785bce6a4

File: pom.xml [hunk 4]
```diff
@@ -110,14 +110,6 @@
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
```
Comment (tblanshard) [inline pom.xml:112]:
```suggestion
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>
        <version>2.7.2</version>
        <configuration>
          <workingDirectory>${project.build.directory}/capstone-project-1</workingDirectory>
        </configuration>
      </plugin>
```

**Review by tblanshard — APPROVED** [review]
LGTM :)

**Pull request merged**